runelite-client/src/main/java/net/runelite/client/plugins/music/MusicPlugin.java

```diff
@@ -466,17 +466,20 @@ private void updateList(Widget musicList, String filter)
 
 		public void update()
 		{
-			handle.setOnDragListener((JavaScriptCallback) this::drag);
-			handle.setOnDragCompleteListener((JavaScriptCallback) this::drag);
-			handle.setHasListener(true);
-
-			track.setOnMouseRepeatListener((JavaScriptCallback) ev ->
+			JavaScriptCallback setSliderTooltip = ev ->
 			{
 				int value = channel.getValue();
 				int percent = (int) Math.round((value * 100.0 / channel.getMax()));
 
 				sliderTooltip = new Tooltip(channel.getName() + ": " + percent + "%");
-			});
+			};
+
+			handle.setOnMouseRepeatListener(setSliderTooltip);
```

Comment:
you can avoid this listener by setting noClickThrough to false

Reply:
Good call; fixed